Doc/library/types.rst

```diff
@@ -198,16 +198,23 @@ Standard names are defined for the following types:
          Defaults to ``None``. Previously the attribute was optional.
 
 
-.. data:: TracebackType
+.. class:: TracebackType(tb_next, tb_frame, tb_lasti, tb_lineno)
```

Comment:
In the `class` directives, things in parens should be base classes, not constructor parameters.

Reply:
@merwok This PR is old enough now that fixing the markup here is likely better handled as a separate issue on the tracker.